### PR TITLE
Add github issue template

### LINKED ISSUE #194: Add github issue template

### PR DESCRIPTION
**Pull request by LucianBuzzo**
Fixes #194

change-type: minor

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+1.
+1.
+1.
+
+## Specifications
+
+- Browser:
+- OS:
+- Environment: <!-- e.g. production, staging, devenv -->
```
Comment (lekkas) [inline .github/ISSUE_TEMPLATE.md]: Is this relevant in pensieve?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+## Expected Behavior
+
+
+## Actual Behavior
+
+
+## Steps to Reproduce the Problem
+
+1.
+1.
```
Comment (lekkas) [inline .github/ISSUE_TEMPLATE.md]: uber nit: should the list be 1. 2. 3. ? I know we approved/merged the UI template with the 1. 1. 1. sequence but I never asked why in that PR 🙃

Comment (LucianBuzzo) [reply .github/ISSUE_TEMPLATE.md]: The 1. 1. 1. sequence just creates an ordered list that goes from 123. for example:

1.
1.
1.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (LucianBuzzo) [reply .github/ISSUE_TEMPLATE.md]: No I don't think it is!

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (lekkas) [reply .github/ISSUE_TEMPLATE.md]: Ah I see, so GH markdown renders 1.1.1. properly to a 123 list , I thought 1.2.3 is *required* in the template to have the 123 render. Thanks, works for me in any case!

Comment (LucianBuzzo) [reply .github/ISSUE_TEMPLATE.md]: https://daringfireball.net/projects/markdown/syntax#list Explains this quite well

**Pull request merged**